design-reviews/trade-studies/readme.md

```diff
@@ -0,0 +1,49 @@
+They evaluate potential choices against a set of objective criteria/customer requirements to clearly lay out the benefits and limitations
+of each solution.
+
+[Trade studies](https://en.wikipedia.org/wiki/Trade_study) are a concept from systems engineering that we adapted for CSE projects. Trade
+studies have proved to be a critical tool to drive alignment with the customers, earn credibility while doing so and ensure our decisions
+were backed by data and not bias.  
+
+## When to use the tool
+
+Trade studies go hand in hand with high level architecture design. This usually occurs as project requirements are solidifying, before
+coding begins. Trade studies continue to be useful throughout the project at any time where there are multiple options that need
+to be selected from. New decision point could occur from changing requirements, getting results of a research spike, or identifying
+challenges that were not originally seen.
+
+Trade studies should be avoided if there is a clear solution choice. Because they require each solution to be fully thought out, they
+have the potential to take a lot of time to complete. When there is a clear design, the trade study should be omitted and an entry
+should be made in the [Decision Log](../decision-log/readme.md) documenting the decision.
+
+## Why Trade Studies
+
+The biggest advantage of using trade studies is formalizing the decision-making process in a clear and objective manner. This gives a clear
+flow for presenting the information to decision makers. We recommend the following approach for sharing information:
+
+|Sharing Approach|Customer Cloud Maturity|Content Presentations|Tools Used fo Writing/Sharing|
+|-|-|-|-|
+|Lightweight|Beginner|Executive summary, Diagrams/visuals, comparison tables | Word doc with referenced Visio diagram and Excel tables, Wiki software with built-in visuals|
+|Full Experience|Intermediate to advanced|All of the above, 30-50% of the trade study text|Same as above|
+
+Additionally, filling out the trade study template provides structure to the design phase and leaves a documentation record of assumptions
+and data that went into each decision.
+
+## Flow of a Trade Study
+
+Trade studies can vary widely in scope; however, they follow the common pattern below:
+
+1. Solidify the requirements – Work with the customer to agree on the requirements for the functionality that you are trying to build.
+1. Create evaluation criteria – This is a set of qualitative and quantitative assessment points that represent the requirements. Taken together, they become an easy to measure stand-in for the potentially abstract requirements.
+1. Brainstorm solutions – Gather a list of possible solutions to the problem. Then, use your best judgement to pick the 2-4 solutions that seem most promising. For assistance narrowing solutions, reach out to Tech Domains, Product Groups and other XDC teams that have faced similar problems.
```

Comment:
Keep in mind that the audience of this content is not limited to MSFT CSE employees. This is intended to be usable by our customers; even if they are not involved directly within CSE engagement. For that last sentence, I would suggest making that more generic so that its applicable in the customer context as well.

Reply:
Good point. I removed all references to CSE specific terms, and changed references to "customer" to "stakeholder" to make it more general